apps/server/src/modules/files-storage/service/files-storage.service.ts

```diff
@@ -316,18 +316,19 @@ export class FilesStorageService {
 		return fileRecord;
 	}
 
-	private sendToAntiVirusService(sourceFile: FileRecord) {
-		if (sourceFile.isPending()) {
-			this.antivirusService.send(sourceFile);
+	private sendToAntiVirusService(fileRecord: FileRecord) {
+		if (fileRecord.isPending()) {
+			this.antivirusService.send(fileRecord.getSecurityToken());
 		}
 	}
 
+	// TODO: should be private
```

Comment:
Must be changed to private and rewrite effected tests

Reply:
And copyFileRecord should be private as well or shouldn't it?